website/docs/cli/commands/docs/docs-generate.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: atmos docs generate
+sidebar_label: generate
+sidebar_class_name: command
+description: "Generate readme subcommand for Atmos docs"
+id: generate
+---
+import Screengrab from '@site/src/components/Screengrab'
+
+:::note Purpose
+
+Use this command to generate a README.md from one or more input sources.  
+
+:::
+
+The sources are defined at `docs.generate.readme` section of `atmos.yaml`.
+
+    ```yaml
+    docs:
+    generate:
+      # Generate README in current working directory
+      readme:
+        base-dir: .
+        input:
+          - "./README.yaml"
+        # A template can be a remote URL/github/gitlab/etc 
+        template:  "https://raw.githubusercontent.com/cloudposse/.github/5a599e3b929f871f333cb9681a721d26b237d8de/README.md.gotmpl"
+        # The final README
+        output: "README.md"
+        terraform:
+          source: src/
+          enabled: false
+          format: "markdown"
+          show_providers: false
+          show_inputs: true
+          show_outputs: true
+          sort_by: "name"
+          hide_empty: false
+          indent_level: 2
+
+    ```
```

Comment:
fix indentation

```suggestion
    ```yaml
    docs:
      generate:
        # Generate README in current working directory
        readme:
          base-dir: .
          input:
            - "./README.yaml"
          # A template can be a remote URL/github/gitlab/etc 
          template:  "https://raw.githubusercontent.com/cloudposse/.github/5a599e3b929f871f333cb9681a721d26b237d8de/README.md.gotmpl"
          # The final README
          output: "README.md"
          terraform:
            source: src/
            enabled: false
            format: "markdown"
            show_providers: false
            show_inputs: true
            show_outputs: true
            sort_by: "name"
            hide_empty: false
            indent_level: 2

    ```
```